centreon/src/App/MonitoringConfiguration/Infrastructure/ApiPlatform/State/ListStandardMacrosProvider.php

```
@@ -0,0 +1,101 @@
+        #[Autowire(service: ResourceStandardMacroTransformer::class)]
+        private TransformerInterface $transformer,
+        private StandardMacroRepository $macroRepository,
+        private Pagination $pagination,
+    ) {
+
+    }
+
+    /**
+     * @return iterable<StandardMacroResource>
+     */
+    public function provide(Operation $operation, array $uriVariables = [], array $context = []): iterable
```

PHPMD (2.15.0)

Avoid unused parameters such as '$uriVariables'. (undefined)

(UnusedFormalParameter)